config/sync/core.entity_form_display.paragraph.components.default.yml

```diff
@@ -24,7 +24,7 @@ content:
       closed_mode_threshold: 0
       add_mode: dropdown
       form_display_mode: default
-      default_paragraph_type: _none
+      default_paragraph_type: rich_text
       features:
         duplicate: duplicate
         collapse_edit_all: collapse_edit_all
```

Comment:
Within the components paragraph settings can you also please update the edit mode to be closed

Reply:
Done

Reply:
Done